tests/testutil.h

```diff
@@ -201,12 +201,12 @@ static void NN_UNUSED test_drop_impl (char *file, int line, int sock, int err)
     }
 }
 
-static int get_test_port(int argc, const char *argv[])
+static int NN_UNUSED get_test_port (int argc, const char *argv[])
```

Comment:
This follows the precedent set by all other functions in this file. I believe it's some artifact of how the macros are called, not these functions themselves.